python/python/Ice/ValueFactory.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) ZeroC, Inc. All rights reserved.
+#  Copyright (c) ZeroC, Inc.
```

Comment:
It also seems like _some_ of the Python files also got an extra space inserted in them.

Not the files which have a shebang, those python files seem fine.
But most of the other ones.